lib/ansible/module_utils/network/ios/config/l2_interfaces/l2_interfaces.py

```diff
@@ -36,6 +36,7 @@ class L2_Interfaces(ConfigBase):
     ]
 
     access_cmds = {'access_vlan': 'switchport access vlan'}
+    voice_cmds = {'voice_vlan': 'switchport voice vlan'}
```

Comment:
Not sure if this should just be moved under `access_cmds` as this is an access port option and I'd be more than happy to change the code to reflect that as well.